src/algorithms/algorithm.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <string>
+
+class Game;
+
+class Algorithm{
```

Comment:
Algorithm is too general name imo. Everything in code is an algorithm

Reply:
Ok